v2/gcd.go

```diff
@@ -26,6 +26,7 @@ package gcd
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
should this be "github.com/goccy/go-json" ?

Reply:
eeep! It should be, cheers. I'll fix it.